### PR TITLE
persistence-client-library: install data folders

### PR DESCRIPTION
**Pull request by sashko**
Persistence expects its data under the two folders /Data/mnt-c and
/Data/mnt-wt.

Make sure those folders are present on the filesystem.

Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

File: recipes-extended/persistence-client-library/persistence-client-library_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,9 @@
+do_install_append() {
+    mkdir -p ${D}/Data/mnt-wt/
```
Comment (fhajredini) [inline recipes-extended/persistence-client-library/persistence-client-library_%.bbappend]: You should use `install -d` instead

File: recipes-extended/persistence-client-library/persistence-client-library_%.bbappend [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    mkdir ${D}/Data/mnt-c/
```
Comment (fhajredini) [inline recipes-extended/persistence-client-library/persistence-client-library_%.bbappend]: Same

**Comment by fhajredini** [comment]
Have you tested this with the changes on #258 ?

**Comment by sashko** [comment]
> Have you tested this with the changes on #258 ?

no, but there shouldn't be any problems :crossed_fingers:

**Comment by fhajredini** [comment]
> > Have you tested this with the changes on #258 ?
> 
> no, but there shouldn't be any problems crossed_fingers

I have merged #258 and gave this another run on Jenkins.

**Pull request merged**